### PR TITLE
Feat/success page

### PR DESCRIPTION
**Pull request by lucasveigaa**

### CODE REVIEW

**Review by orloke — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+	<link rel="icon" type="image/svg+xml" href="/src/assets/favicon.svg" />
+	<meta name="theme-color" content="#49D13A" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap" rel="stylesheet" />
+	<title>Gama Academy</title>
+</head>
+
+<body>
+	<div id="root"></div>
+	<script type="module" src="/src/main.tsx"></script>
```
Comment (andresdslima) [inline index.html]: '/src/index.tsx'

File: src/components/SuccessPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Button } from '@gama-academy/smash-web';
+import Cookies from 'js-cookie';
+import illustration from '../../assets/illustration-page-success.png';
```
Comment (andresdslima) [inline src/components/SuccessPage/index.tsx]: '../../assets/images/illustration-page-success.png'

**Review by andresdslima — COMMENTED** [review]
so essas 2 correcoes, eu acho...

**Review by andresdslima — APPROVED** [review]
lgtm

**Pull request merged**